test/integ/test_timestamp_binding.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from datetime import datetime
+from typing import List
+
+import pytest
+
+from snowflake.connector.cursor import SnowflakeCursor
+
+from ..randomize import random_string
```

Comment:
this doesn't exist in old driver, but if you consolidate this test into test_binding.py I don't think it's an issue there